### PR TITLE
Tests refactoring

### PR DESCRIPTION
**Pull request by NikPaukov**

### CODE REVIEW

**Review by Kivanval — APPROVED** [review]

File: src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import com.qunite.api.service.UserService;
+import com.qunite.api.web.mapper.QueueMapper;
+import java.util.Random;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
```
Comment (xd6d) [inline src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java:29]: optimize imports

File: src/test/java/com/qunite/api/web/QueueControllerTest.java [hunk 2]
```diff
@@ -60,21 +61,20 @@ void retrieveByIdWhenExists() throws Exception {
...
     given(queueService.findById(anyLong())).willReturn(Optional.of(queue));
 
-    var resultActions = mockMvc.perform(get(url + "/" + queue.getId())
-        .accept(MediaType.APPLICATION_JSON));
+    var resultActions = mockMvc.perform(get("/{url}/{id}", url, queue.getId()));
+
     resultActions.andExpect(status().isOk())
-        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
-        .andExpect(content().json("{id: 1}"));
+        .andExpect(jsonPath("$.id", is(queue.getId().intValue())));
```
Comment (xd6d) [inline src/test/java/com/qunite/api/web/QueueControllerTest.java:67]: why did you replace 
.andExpect(content().json("{id: 1}"));
with .intValue()?

Comment (xd6d) [reply src/test/java/com/qunite/api/web/QueueControllerTest.java:67]: or maybe rewrite .intValue with .andExpect(jsonPath("$.id", is(1))); ?

File: src/test/java/com/qunite/api/web/QueueControllerTest.java [hunk 3]
```diff
@@ -160,18 +155,19 @@ void createQueue() throws Exception {
...
-    var resultActions = mockMvc.perform(post(url)
+
+    var resultActions = mockMvc.perform(post("/{url}", url)
         .contentType(MediaType.APPLICATION_JSON)
         .content(json));
+
     resultActions.andExpect(status().isCreated())
-        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
-        .andExpect(content().json("{id: 1}"));
+        .andExpect(jsonPath("$.id", is(queue.getId().intValue())));
```
Comment (xd6d) [inline src/test/java/com/qunite/api/web/QueueControllerTest.java:164]: same is above

File: src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  }
+
+  @Test
+  @Sql({"/users-create.sql", "/queues-create.sql"})
+  void deleteQueue() throws Exception {
+    var queueId = new Random().nextLong();
+
+    mockMvc.perform(delete("/{url}/{id}", url, queueId))
+        .andExpect(status().isNoContent());
+  }
```
Comment (xd6d) [inline src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java:178]: Does not look like the right way to test deletion

**Review by xd6d — CHANGES_REQUESTED** [review]
fix all .intValue() and code smells

File: src/test/java/com/qunite/api/web/QueueControllerTest.java (hunk 3 above)
Comment (NikPaukov) [reply src/test/java/com/qunite/api/web/QueueControllerTest.java:163]: same as above

**Review by xd6d — APPROVED** [review]

**Pull request merged**